### DISCUSSION TITLE
Requests for an export function

### DISCUSSION THREAD

**Discussion by teddythinh** [superlinked/VectorHub#363] [Discussions]
Hi team, I'm really interested in your project. I would like to request a feature that I can export the table to a `.csv` or `.xslx` file. Thank you!

**Comment by svonava** [comment]
Hi! To better understand your use case, could you elaborate on what you are trying to achieve with the export? Thanks!

**Reply by teddythinh** [reply]
Thanks for the fast reply. I would use it to make some visualizations or to analyse the data such as EDA.

**Reply by svonava** [reply]
Got it, would you be open to contribute the analysis back to hub.superlinked.com?

The underlying data is open source here: https://github.com/superlinked/VectorHub/tree/main/docs/tools/vdb_table/data

You could read all the files, concatenate them and convert to a dataframe to get started - example code:
```python
import os
import json
import pandas as pd

def load_jsons_to_dataframe(folder_path):
    # List to hold all JSON objects
    data_list = []

    # Loop through each file in the specified folder
    for filename in os.listdir(folder_path):
        if filename.endswith('.json'):  # Check if the file is a JSON file
            file_path = os.path.join(folder_path, filename)
            with open(file_path, 'r') as file:  # Open the file
                data = json.load(file)  # Load the JSON data
                data_list.append(data)  # Append to the list

    # Convert the list of JSON objects into a DataFrame
    dataframe = pd.DataFrame(data_list)
    return dataframe

# Usage example:
# df = load_jsons_to_dataframe('path/to/your/folder')
# print(df)
```

**Reply by teddythinh** [reply]
Hi there, thanks for the provided information. I just need a function that allows a non-code user to export the table to the desired format such as an Excel table. I believe it would be easy because Pandas supports many options to convert into a file such as `to_csv`, `to_excel`, `to_json`,...

Also thank you for pointing up the underlying data, I will dive into it and get back to you.
